src/core/InputField/__tests__/InputField.test.tsx

```diff
@@ -0,0 +1,22 @@
+import { render } from '@testing-library/react';
+import React from 'react';
+import { KuberaThemeProvider } from '../../../theme';
+import { InputField } from '../InputField';
+
+describe('InputField', () => {
+  it('Renders', () => {
+    const { getByTestId } = render(
+      <KuberaThemeProvider platform="kubera-chaos">
+        <InputField
+          label="primary"
+          variant="primary"
+          type="text"
+          disabled={false}
+          handleChange={() => console.log('change')}
+        />
+      </KuberaThemeProvider>
+    );
+
+    expect(getByTestId('inputField')).toBeTruthy();
```

Comment:
Test cases should be like...
If I type a text...is it showing or not...
If it's password will it show
If I click on visibility button will it show
etc etc